e2e/tests/121-Portal/ViewPaProfilePage/OpenProfilePage.spec.ts

```diff
@@ -47,9 +48,8 @@ test('[27411] Open PA profile page', async ({ page }) => {
       firstName: undefined,
       lastName: undefined,
       phoneNumber: undefined,
-      status: 'Registered',
+      status: undefined,
```

Comment:
Why is this undefined here? I think this could refer to the en.json on this line: https://github.com/global-121/121-platform/blob/0c957878576866bd5706817d7dc6c6850842226c/interfaces/Portal/src/assets/i18n/en.json#L66. Is that possible?

Reply:
I did it on purpose here to avoid conflicts since I wanted to check only the first cell of the row to save time as main objective of the test does not refer to check the table. When you put undefined then function does not search for the value. But maybe it can be done better.

Reply:
I'm open for suggestions ;)

Reply:
But I also skipped this part because it is not relevant for the test. I mapped the table and added this validation for the future use and it most certainly can become much better.